src/features/compile/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { isArray } from '@umijs/utils/compiled/lodash';
+import { IApi } from 'umi';
+
+export const shouldDisabledLiveDemo = (api: IApi) => {
+  const extraBabelPlugins = api.userConfig.extraBabelPlugins;
+  const disableFlag =
+    isArray(extraBabelPlugins) &&
+    extraBabelPlugins?.some((p: any) =>
+      /^import$|babel-plugin-import/.test(p[0]),
+    );
+  if (disableFlag) {
+    api.logger.warn(
+      "dumi don't suggest to use babel-plugin-import, liveDemo will be disabled because of this config.",
```

Comment:
感觉一两句话说不清楚，搞个 FAQ 解释下？主要传达两点信息：
1. dumi 的 live demo 特性与 babel-plugin-import 存在冲突，所以使用后者时前者会自动禁用
2. 如果开发者希望使用 live demo，那么建议去掉 babel-plugin-import，如果需要按需加载样式，那么使用 `.dumi/global.css` 引入全量样式

命令行提示类似 `live demo feature has been automatically disabled since babel-plugin-import be registered, if you want to enable live demo feature, checkout: https://xxxx`

以及直接用 logger 不要用 api.logger，不然前缀会很长